src/Microsoft.AspNetCore.WebHooks.Common/Extensions/DictionaryExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel;
+
+namespace System.Collections.Generic
```

Comment:
Why are these exposed as public API on types we don't own? This should be shared source

Reply:
Absolutely. This was straight out of `Microsoft.AspNet.WebHooks.Common` and I didn't plan to ship with the extension methods in place.

Could skip creating a shared-source package and make these non-extension `static` methods in a `Microsoft.AspNetCore.WebHooks` namespace. The methods are used in various receivers but I'm not sure user code (controller actions in the new world) need them.